emperor/support_files/js/shapes.js

```diff
@@ -0,0 +1,17 @@
+define(['jquery', 'three'], function($, THREE) {
+
```

Comment:
Can you add a comment block of what this module is intended for i.e. to avoid the duplication of the source between controller and editor/formatter.